dallinger/experiment_server/experiment_server.py

```diff
@@ -792,6 +792,27 @@ def get_participant(participant_id):
     return success_response(participant=ppt.__json__())
 
 
+@app.route("/participant", methods=["POST"])
+def load_participant():
+    """Get the participant with an assignment id provided in the request.
+    Delegates to :func:`~dallinger.experiments.Experiment.load_participant`.
+    """
+    assignment_id = request_parameter("assignment_id", optional=True)
+    if assignment_id is None:
+        return error_response(
+            error_type="/participant GET: no participant found", status=403
```

Comment:
- Looks like this route actually accepts only POST requests(?) 
- Should the error be a 404?

Reply:
There's some discussion on the ticket regarding whether the route might change DB state in some implementations. Because that's a possibility (though it's not true in the default implementation), I decided to use `POST`. That also allows us to stick with the simple route names like `/participant` overloaded based on arguments/request type. Not sure if it's an ideal choice, but it seems to work well enough.

I think it could be a 404 or 400 (Bad Request when the param isn't included), but the `/participant` `GET` route returns `403` for missing participants so I thought it would be a good idea to stay consistent with that.

Reply:
I guess then maybe the error message should be updated to say 'POST'?

Reply:
POST definitely sounds right, and I agree consistency with the other route makes sense, so yeah, I think we just fix the error message.

Reply:
Oh yeah. Oops. Fixed.